sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1651,11 +1651,11 @@ export enum SasTokenPermissionKind {
     // (undocumented)
     TriggerReplace = 33554432,
     // (undocumented)
-    UserDefinedFuntionDelete = 16777216,
+    UserDefinedFunctionDelete = 16777216,
```

Comment:
would it be possible to keep the old typo'd version with an `@deprecated` comment?